Net12/WebMaze/Views/Shared/_Layout.cshtml

```diff
@@ -64,6 +64,10 @@
                     <li class="nav-item">
                         <a class="nav-link" href="/CellInfo/AddCell">Add Cell</a>
                     </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/Maze/Index">Maze</a>
```

Comment:
Mae be <a class="nav-link" href="/Maze/ManageMazeDifficult">ManageMazeDifficult</a>

Reply:
no